client/src/components/Error.res

```diff
@@ -1,3 +1,6 @@
+// Defines an Error message type and related functions
+// Errors are currently represented as a simple option<string>
```

Comment:
Not really. This is more of the start of trying to pull out the error bar as a component.